.github/workflows/e2e.yaml

```diff
@@ -20,29 +20,16 @@ jobs:
       run: ./mongoseed.sh
       working-directory: ./database
 
-    - name: Cache node modules
-      uses: actions/cache@v1
-      with:
-        path: ~/.npm
-        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-        restore-keys: |
-          ${{ runner.os }}-node-
-
-    - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: 12.x
-
-    - name: Install dependencies (npm ci)
-      run: npm ci
-      working-directory: ./client
-
     - name: Set up JDK 1.8
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
-
-    - name: Start server and run e2e tests
-      run: cd server && ./gradlew run & cd client && npm run e2eci
+        
+    - name: Run e2e tests
+      uses: cypress-io/github-action@v2
+      with:
+        start: ${{ github.workspace }}/server/gradlew run -p ${{ github.workspace }}/server/
+        command: npm run e2e
```

Comment:
It could very well be `npx ng e2e`. The reason for using a npm script like this is that it allows you to specify additional parameters in the `package.json` which is good if the command is used in multiple places (eg if you had multiple CI scripts) or if you want to use the npm `before` and `after` hooks.

Reply:
Are we using that feature here? Do we think we're likely to? If it's clearly useful, then we can certainly use `npm run` instead, but if we're not likely to to then why not use `npx ng` instead. Can we include comments in the YAML file (not sure if the syntax allows that) to "remind" people that they can switch from `npx ng` to `npm run` if we want/need to put things in `package.json`?